home-manager/packages.nix

```diff
@@ -44,8 +44,8 @@
     gh
     ghq
 
-    # GPG
     gnupg
+    edge-pkgs.sequoia-sq # Alt `gpg` - nixos-24.05 does not backport recent versions and the older requires to rebuild. https://github.com/NixOS/nixpkgs/pull/331099
```

Comment:
For context of replacing gpg, I need https://gitlab.com/sequoia-pgp/sequoia-chameleon-gnupg.

So waiting to get binary cache of https://github.com/NixOS/nixpkgs/pull/349002

Reply:
Updated commit has been merged in https://github.com/kachick/dotfiles/pull/849/commits/95cf173855f3bf39cdb7a06133009b6f5fc1d76d